docs/en/advanced-topics/boot-mode-selection.rst

```diff
@@ -164,6 +164,7 @@ Manual Bootloader
 Depending on the kind of hardware you have, it may also be possible to manually put your {IDF_TARGET_NAME} board into Firmware Download mode (reset).
 
 - For development boards produced by Espressif, this information can be found in the respective getting started guides or user guides. For example, to manually reset a development board, hold down the **Boot** button (``{IDF_TARGET_STRAP_BOOT_GPIO}``) and press the **EN** button (``EN`` (``CHIP_PU``)).
+- Note: Only release **Boot** button when connection is established.
```

Comment:
This is not entirely true. It is not needed to enter Download mode, just to stay there if for some reason esptool's reset sequence on connection doesn't work.

I think we could add a note as the last item stating that this in practice means to hold the boot button or pull the boot pin down while starting esptool (with the default reset sequence) and keeping this until esptool establishes connection.